src/app/components/game/capturedTable/CapturedTable.ts

```diff
@@ -0,0 +1,34 @@
+import styles from './CapturedTable.module.scss';
+
+type Piece = 'queen' | 'king' | 'pawn' | 'knight' | 'rook' | 'bishop';
+
+type CapturedPiecesSide = 'player' | 'opponent';
+
+export class CapturedTable {
+    private __element: HTMLDivElement;
+
+    constructor(side: CapturedPiecesSide, capturedPieces: Piece[]) {
+        this.__element = document.createElement('div');
+        this.__element.classList.add(styles.capturedWrapper);
+        this.__element.classList.add(styles[side]);

```

Comment:
Możesz zapisać kilka klas po przecinku, czyli `this.__element.classList.add(styles.capturedWrapper, styles[side]);`